src/eca/features/tools.clj

```diff
@@ -19,38 +19,75 @@
 
 (def ^:private logger-tag "[TOOLS]")
 
+;; ============================================================================
```

Comment:
please let's avoid changing a lot of stuff here:
- those comment headers are unecessary
- focus on adding the tool in the concat like I mentioned